azext_iot/tests/iothub/devices/hierarchy_configs/invalid/duplicate_device_config.yml

```diff
@@ -0,0 +1,16 @@
+config_version: "1.0"
+# We don't need iothub info from the config schema
```

Comment:
We talked about exposing one or more markdown document at least in our project docs that shows or links to example(s) of the config file format and describes key usage information.

To do this, we should concretely pin the structure of the format and version it. Best practice conventions should be applied, and because the CLI supports either json or yaml, I don't think its a bad idea to align on camelCase since that is an accepted practice for both json and yaml.